src/targets/github.ts

```diff
@@ -422,6 +334,25 @@ export class GithubTarget extends BaseTarget {
     }
   }
 
+  /**
+   * Publishes the release draft.
+   *
+   * @param release_id Release object ID
+   */
+  public async publishReleaseDraft(release_id: number) {
+    if (isDryRun()) {
+      this.logger.info(`[dry-run] Not publishing the release draft`);
+      return;
+    }
+
+    await this.github.repos.updateRelease({
+      owner: this.githubConfig.owner,
+      repo: this.githubConfig.repo,
```

Comment:
```suggestion
      ...this.githubConfig,
```

Reply:
Done in 7a968f67abeba6dadb462224bba5b1ed4f6fa1dc. 👍